content/04.study.md

```diff
@@ -79,60 +79,64 @@ Moreover, these nonlinear algorithms can deconvolve the effects of multiple vari
-Lanchantin et al. [@tag:Lanchantin2016_motif] introduced several new convolutional and recurrent neural network models that further improved TFBS predictive accuracy.
-Due to the motif-driven nature of the TFBS task, most architectures have been convolution-based [@tag:Zeng2016_convolutional].
-While many models for TFBS prediction resemble computer vision and NLP tasks, it is important to note that DNA sequence tasks are fundamentally different.
-Thus the models should be adapted from traditional deep learning models in order to account for such differences.
-For example, motifs may appear in either strand of a DNA sequence, resulting in two different forms of the motif (forward and reverse complement) due to complementary base pairing.
-To handle this issue, specialized reverse complement convolutional models share parameters to find motifs in both directions [@tag:Shrikumar2017_reversecomplement].
-
-Despite these advances, several challenges remain.
-First, because the inputs
-(ChIP-seq measurements) are continuous and most current algorithms are designed to produce binary outputs (whether or not there is TF binding at a particular site), false positives or false negatives can result depending on the threshold chosen by the algorithm.
-Second, most methods predict binding of TFs at sites in isolation, whereas in reality multiple TFs may compete for binding at a single site or act synergistically to co-occupy it.
-Fortunately, multi-task models are rapidly improving at simultaneous prediction of many TFs' binding at any given site [@tag:Zhou2015_deep_sea].
-Third, it is unclear exactly how to define a non-binding or "negative" site in the training data because the number of positive binding sites of a particular TF is relatively small with respect to the total number of base-pairs in a genome (see Discussion).
-
-While deep learning-based models can automatically extract features for TFBS prediction at the sequence level, they often cannot predict binding patterns for cell types or species that have not been previously studied (i.e. that we don't have labelled data for).
-
-For many cell types, we often have experimental TF binding data for a certain cell type (e.g. normal), but not the one of interest (e.g. cancer).
-To handle this issue, there are two options.
-The first is to use features such as epigenomic marks which are specific to the certain cell type.
-If we have these features available, this is probably the best approach for accurate prediction since they will be strong signals for that particular cell type and TF.
-The second option is to use domain adaptation, which is a type of transfer learning where we learn a model which can predict from features in one domain (e.g. normal) and transfer the model to predict on features in another domain (e.g. cancer).
-This has been done in many other areas such as sentiment analysis [@tag:Glorot2011_domain].
```

Comment:
`tag:Glorot2011_domain` was replaced with `arxiv:1502.02791; arxiv:1505.07818` for domain adaptation.  That seems okay to me.